go.mod

```diff
@@ -213,6 +213,8 @@ require (
 	github.com/grafana/grafana/pkg/storage/unified/resource v0.0.0-20240821161612-71f0dae39e9d // @grafana/grafana-search-and-storage
 )
 
+require github.com/grafana/authlib/claims v0.0.0-20240814074258-eae7d47f01db
```

Comment:
I think if you update to latest for authlib you can get rid of this one

Reply:
it was a dangling reference from https://github.com/grafana/grafana/pull/99244 🤦🏻